conn.go

```diff
@@ -44,14 +66,15 @@ type Conn struct {
 	isStartingTLS       bool
 	Debug               debugging
 	chanConfirm         chan bool
-	chanResults         map[int64]chan *ber.Packet
+	chanResults         map[int64]chan *PacketResponse
 	chanMessage         chan *messagePacket
 	chanMessageID       chan int64
 	wgSender            sync.WaitGroup
 	wgClose             sync.WaitGroup
 	once                sync.Once
 	outstandingRequests uint
 	messageMutex        sync.Mutex
+	RequestTimeout      time.Duration
```

Comment:
rather than exporting this field, can we add a setter to the exported interface?